Src/VimCore/SelectionChangeTracker.fs

```diff
@@ -98,43 +94,6 @@ type internal SelectionChangeTracker
         else
             x.SetModeForSelection()
 
-    /// If the caret changes position and it wasn't initiated by VsVim then we should be 
-    /// adjusting the screen to account for 'scrolloff'

```

Comment:
How do we account for non-VsVim caret movements now?

Reply:
NM, found that this function moved.

---
In reply to: [287606028](https://github.com/VsVim/VsVim/pull/2547#discussion_r287606028) [](ancestors = 287606028)

Reply:
Right, if the selection change tracker were called ExternalEventHandler then it would have made sense to keep the handling of external selection events and external caret events together. As it is, the selection change tracker now only tracks selection changes.